gnosis/safe-cmm-app-react
# Feature/#155/apr

# Description

Adding APR to expanded active strategy view.
Adding tooltips to all fields (to be reviewed by product owners).

Also, big refactor on how prices are fetched.

![screenshot_2020-11-17_15-28-02](https://user-images.githubusercontent.com/43217/99462864-67450680-28e9-11eb-9397-252cccda96ae.png)


# To Test
1. Make sure you have at least one active strategy
1. Go to strategies tab
1. Expand one strategy

- [ ] On the active tab, there should be 4 fields: 
  - [ ] Total value
  - [ ] Hold value
  - [ ] ROI
  - [ ] APR
- [ ] Each field should be populated (in some cases, the fields will have `N/A` instead of a value, and that is expected)

# Background

- Fetching GP historical prices for APR calculation
- Caching the historical prices indefinitely.
- Since GP prices are an average of both sides, using a single cache key for both sides. E.g.: `priceGP(A, B) === priceGP(B, A)`
